### ISSUE TITLE
image path duplicated on multiple instances

### ISSUE THREAD

**Issue by beck24** [websanova/wPaint#100]
My scenario: I'm running wPaint in a lightbox, when the lightbox opens it calls the init functions for wPaint.
On the first opening all works great.  However if the lightbox is closed, then reopened, the menu icons are missing.  Debugging this has lead me to the conclusion that the path setting is appended for each time wPaint is initiated.
See screenshot

The path is /mod/draw/vendors/wPaint/
This is correct on the first instance, on the second instance it tries to retrieve images from /mod/draw/vendors/wPaint//mod/draw/vendors/wPaint/...
Third and fourth inits continue to append the base path setting

![screen shot 2014-12-13 at 10 41 41 pm](https://cloud.githubusercontent.com/assets/738363/5426677/c1d96cb6-8319-11e4-915f-17cb34ad51d2.png)

**Comment by beck24** [comment]
Thanks for a fantastic library by the way - I was using raphael and am switching over to this due to the better default functionality and ease of use!

**Comment by beck24** [comment]
Oh, looks like there are already PR's to address this, just no tickets as far as I can tell

https://github.com/websanova/wPaint/pull/87/files

**Comment by RabindraSharma** [comment]
hello sir can we add more functionality in this plugin like, getting all shapes value in json format when creating shapes as ovals , rectangles, circle and strings values along with this and then retrieve that exact values of letf, top, width, height and colors values
or and also can be editable those component is it possible. please revert me.

**Comment by ketanvekariya** [comment]
Yes you can add more functionality on that like adding custom shapes as per your requirement.
